### ISSUE TITLE
2.3.6 patch release plan

### ISSUE THREAD

**Issue by joeparsons** [az-digital/az_quickstart#1738]
- #1737
- #1708
- #1727
- #1746

**Comment by joeparsons** [comment]
## Pick list
```
git cherry-pick c94ed2a7bcfb6ba90508f80f1dd1d73248fa60fb
git cherry-pick 7da82deb5553607b12b6449be5466af98678666e
git cherry-pick afc3260b50dd7a721426992c7310e7bf6a5513ae
```

**Comment by joeparsons** [comment]
## Release notes snippet

`git log 2.3.5..HEAD --pretty='format:- %s'`

```
- Closes #1745: Use correct full-width setting name when adding full-width library. (#1746)
- Closes #1697: Fix JS error when Text with Background, Text on Media are not full-width (#1727)
- Fixes #1698: Fix sidebar position when full-width content in other regions (#1708)
- Closes #1736: Update Drupal core to 9.3.18. (#1737)
- #1703 2.4.x release preparations (#1706)
```

**Comment by joeparsons** [comment]
Release created: https://github.com/az-digital/az_quickstart/releases/tag/2.3.6
